src/FileGetContents.php

```diff
@@ -198,6 +266,8 @@ protected function getTlsStreamContextDefaults($cafile)
      * @throws \RuntimeException If https proxy required and OpenSSL uninstalled
      *
      * @return resource Default context
+     *
+     * @final since 1.1.0
      */
     protected function getMergedStreamContext($url)
```

Comment:
the function has changed since this has been pasted, see https://github.com/composer/composer/blob/master/src/Composer/Util/StreamContextFactory.php, so maybe we should update the code to reflect on that?